test/unit/BPool/BPool.tree

```diff
@@ -20,3 +20,84 @@ BPool::isBound
 BPool::getNumTokens
 └── when called
     └── it returns number of tokens
+
+BPool::getFinalTokens
```

Comment:
it would be comfy if the order of the function tree definitions matched the order in which functions are defined in the contract. Not a blocker tho.

Reply:
it'd add a lot of changes (changing tree + changing test methods), i think we can leave this as is for now (less merge conflicts)